src/constants.js

```diff
@@ -60,3 +60,4 @@ export const RESET_MODE = ~(
 export const INHERITED_MODES = MODE_HYDRATE | MODE_MUTATIVE_HYDRATE | MODE_SVG;
 
 export const EMPTY_ARR = [];
+export const UNDEFINED = undefined;
```

Comment:
```suggestion
export let UNDEFINED;
```

Reply:
<img width="857" alt="Screenshot 2021-06-10 at 10 00 01" src="https://user-images.githubusercontent.com/17125876/121487703-a1cfd700-c9d2-11eb-8148-be78f6a5602a.png">

Had a pretty bad impact on both size and perf